gogo/jline/src/main/java/org/apache/felix/gogo/jline/Posix.java

```diff
@@ -822,11 +840,26 @@ public void exit() throws Exception {
                 terminal.close();
             }
         };
+
+        // Register a signal handler for INT signal to properly propagate interruption
+        Terminal.SignalHandler prevIntHandler = terminal.handle(Terminal.Signal.INT, signal -> {
```

Comment:
is `Terminal.Signal.INT` working cross-platform? E.g. also on Windows? I know Windows can sometimes be different in handling signals.

Reply:
It definitely is.